pelican/tools/templates/fabfile.py.in

```diff
@@ -3,7 +3,10 @@ import fabric.contrib.project as project
 import os
 import shutil
 import sys
-import SocketServer
+if sys.version_info[0] >= 3:
```

Comment:
I think the `try-except` idiom is [preferred over](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection) doing explicit version detection:

```python
try:
    import socketserver
except ImportError:
    import SocketServer as socketserver
```